packages/models/src/store/store.ts

```diff
@@ -76,6 +86,8 @@ export class Store<
   }
 
   private deserializeCell({ cell, witness }: UpdateStorageValue): GetStorageStruct<StructSchema> {
+    if (!this.schemaOption || typeof this.schemaOption !== 'object' || this.schemaOption == null)
```

Comment:
`this.schemaOption == null` is redundant because `typeof this.schemaOption !== 'object'` filter `null` out